app/controllers/patients_controller.rb

```diff
@@ -265,6 +265,52 @@ def update_hoh
     current_user_patients.where(id: patients_to_update).update_all(responder_id: new_hoh_id)
   end
 
+  def bulk_update_status
+    redirect_to(root_url) && return unless current_user.can_edit_patient?
+
+    patient_ids = params.permit(ids: [])[:ids]
```

Comment:
should probably use `params.require` instead of `params.permit` here since patient_ids should always be required